gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -42,5 +190,13 @@ def extract_output(self, merged_headers):
 class MergeHeaders(beam.PTransform):
   """A PTransform to merge VCF file headers."""
 
+  def __init__(self, split_alternate_allele_info_fields=True):
+    super(MergeHeaders, self).__init__()
+    self._split_alternate_allele_info_fields = (
+        split_alternate_allele_info_fields)
+    self._resolver = HeaderConflictResolver(split_alternate_allele_info_fields)
```

Comment:
Please also remove self._resolver (it'll be inside header_merger). You may still have a local variable for it.

Reply:
Done